src/nvim/normal.c

```diff
@@ -2145,23 +2147,39 @@ void do_check_scrollbind(bool check)
   old_leftcol = curwin->w_leftcol;
 }
 
+static int get_scrollbind_amount(win_T *ref_wp, int ref_old_topline, int topfill_diff)
+{
+  int ref_orig_topline = ref_wp->w_topline;
```

Comment:
Hmm, I meant it's not clear that "orig" is newer than old. There is still an "orig" here.

Reply:
Sorry, that's meant to say `ref_topline`.